Objects/descrobject.c

```diff
@@ -72,22 +72,16 @@ wrapperdescr_repr(PyWrapperDescrObject *descr)
 }
 
 static int
-descr_check(PyDescrObject *descr, PyObject *obj, PyObject **pres)
+descr_check(PyDescrObject *descr, PyObject *obj)
 {
-    if (obj == NULL) {
```

Comment:
Wait, this fixed the bug I was talking about 😆 . Maybe this needs a bpo now?

Reply:
@Fidget-Spinner Thanks! I create the issue about this :)
https://bugs.python.org/issue45385